tests/test_utils.py

```diff
@@ -247,13 +246,11 @@ def test_summary_statistics_are_indentical_to_hardies_paper_confirming_correct_a
     # see http://brucehardie.com/papers/rfm_clv_2005-02-16.pdf
     # RFM and CLV: Using Iso-value Curves for Customer Base Analysis
     df = pd.read_csv('lifetimes/datasets/CDNOW_sample.txt', sep='\s+', header=None, names=['_id', 'id', 'date', 'cds_bought', 'spent'])
-    if sys.version_info[0] < 3:
-        df['date'] = pd.to_datetime(df['date'].astype(unicode))
-    else:
-        df['date'] = pd.to_datetime(df['date'], format='%Y%m%d')
+    df['date'] = pd.to_datetime(df['date'], format='%Y%m%d')
```

Comment:
I still don't understand why there should be errors or what wrong with that change. Tested with python2 and python3:

```python
import pandas as pd
df = pd.read_csv('lifetimes/datasets/CDNOW_sample.txt', sep='\s+', header=None, names=['_id', 'id', 'date', 'cds_bought', 'spent'])
df.head()
"""
	_id 	id 	date 	cds_bought 	spent
0 	4 	1 	19970101 	2 	29.33
1 	4 	1 	19970118 	2 	29.73
2 	4 	1 	19970802 	1 	14.96
3 	4 	1 	19971212 	2 	26.48
4 	21 	2 	19970101 	3 	63.34
"""

pd.to_datetime(df['date'], format='%Y%m%d').head()
"""
0   1997-01-01
1   1997-01-18
2   1997-08-02
3   1997-12-12
4   1997-01-01
Name: date, dtype: datetime64[ns]
"""
```